admin/load.php

```diff
@@ -636,3 +639,40 @@ function perflab_plugin_admin_notices() {
 	}
 }
 add_action( 'admin_notices', 'perflab_plugin_admin_notices' );
+
+/**
+ * Callback function that print plugin activation script.
+ *
+ * @since n.e.x.t
+ */
+function perflab_print_plugin_activation_script() {
+	ob_start();
+	?>
+	( function( $ ) {
+		$( document ).ajaxComplete( function( event, xhr, settings ) {
+			// Check if this is the 'install-plugin' request.
+			if ( settings.data && typeof settings.data === 'string' && settings.data.includes( 'action=install-plugin' ) ) {
+				var target_element = $( event.target.activeElement );
+				// Bail early if the element is not a[href].
+				if ( ! target_element.is( 'a[href]' ) ) {
+					return;
+				}
+				/*
+				 * WordPress core uses a 1s timeout for updating the activation link,
+				 * so we set a 1.5 timeout here to ensure our changes get updated after
+				 * the core changes have taken place.
+				 */
+				setTimeout( function() {
+					var plugin_url = target_element.attr( 'href' );
```

Comment:
That check here is not what I meant in https://github.com/WordPress/performance/pull/900#discussion_r1424303670. It's safer to check that `plugin_url` is actually set. For example:
```suggestion
				if ( ! target_element ) {
					return;
				}
				/*
				 * WordPress core uses a 1s timeout for updating the activation link,
				 * so we set a 1.5 timeout here to ensure our changes get updated after
				 * the core changes have taken place.
				 */
				setTimeout( function() {
					var plugin_url = target_element.attr( 'href' );
					if ( ! plugin_url ) {
						return;
					}
```